src/types/mod.rs

```diff
@@ -587,6 +587,16 @@ impl JsObject {
             Handle::new_internal(JsObject(local))
         }
     }
+
+    #[cfg(feature = "napi-8")]
+    pub fn freeze<'a, C: Context<'a>>(self, cx: &mut C) -> bool {
+        unsafe { neon_runtime::object::freeze(cx.env().to_raw(), self.to_raw()) }
+    }
+
+    #[cfg(feature = "napi-8")]
+    pub fn seal<'a, C: Context<'a>>(self, cx: &mut C) -> bool {
```

Comment:
I think it would be better to add `freeze` and `seal` to the [`Object` trait](https://github.com/neon-bindings/neon/blob/7c93fd8d4e68a277026707d734b42eca02c20e3d/src/object/mod.rs#L92) instead. That way they can be called from anything that is an `Object`, instead of needing to be upcast to a `JsObject` first.